src/sessions/softphone-session-handler.ts

```diff
@@ -564,22 +555,15 @@ export default class SoftphoneSessionHandler extends BaseSessionHandler {
       const terminatedPromise = new Promise<JingleReason>((resolve) => {
         const listener = (endedSession: IExtendedMediaSession, reason: JingleReason) => {
           if (endedSession.id === session.id && endedSession.conversationId === conversationId) {
-            this.log('debug', 'received "sessionEnded" event from session requested by `sdk.endSession()`', {
-              endedSession,
-              conversationId,
-              session
-            }, { skipServer: true });
+            this.log('debug', 'received "sessionEnded" event from session requested by `sdk.endSession()`', { endedSession, conversationId, session}, { skipServer: true });
             this.sdk.off('sessionEnded', listener);
             return resolve(reason);
           } else {
-            this.log('debug', 'received "sessionEnded" event from session that was NOT requested by `sdk.endSession()`', {
-              endedSession,
-              conversationId,
-              session
-            }, { skipServer: true });
+            this.log('debug', 'received "sessionEnded" event from session that was NOT requested by `sdk.endSession()`', { endedSession, conversationId, session}, { skipServer: true});
           }
         }
 
+        this.log('info', 'received "sessionEnded event"', { sessionId: session.id, conversationId, sessionType: session.sessionType})
```

Comment:
I actually want to change this. The `endSession` fn only gets called when *this client* ends the call. In the base-session we are logging a message like this:
```
    this.log('info', 'ending session', { sessionId: session.id, sessionConversationId: session.conversationId, reason, sessionType: session.sessionType });
```
but we aren't doing that for the softphone handler. Let's also add this message at the top of this fn. Then, I want the timeline to always have a `sessionEnded` event even if it's ended remotely. It looks like all we'd have to do is something like this:
```
this.sdk.on('sessionEnded', (session, reason) => {
  log('sessionEnded event received', { sessionId: session.id, conversationId: session.conversationId, sessionType, reason };
})
```
We just need an appropriate place for it, which should probably be generic so let's put it in the base-session-handler.

tldr; 
* add a endingSession message at the top of softphone-handler's `endSession()` fn.
* add a "received sessionEnded event" message in the base-session-handler.